website/docs/api/themes/theme-configuration.md

```diff
@@ -208,6 +208,25 @@ You can add items to the navbar via `themeConfig.navbar.items`.
 
 By default, Navbar items are regular links (internal or external).
 
+React Router should automatically apply active link styling to links, but you can use `activeBasePath` in edge cases. For cases in which a link should be active on several different paths (such as when you have multiple doc folders under the same sidebar), you can use `activeBaseRegex`. `activeBaseRegex` is a more flexible alternative to `activeBasePath` and takes precedence over it -- Docusaurus parses it into a regular expression that is tested against the current URL.
+
+Outbound (external) links automatically get `target="_blank" rel="noopener noreferrer"` attributes.
+
+Accepted fields:
+
+| Field | Value | Explanation | Required | Default |
+| --- | --- | --- | --- | --- |
+| `label` | `string` | The name to be shown for this item. | Yes | N/A |
+| `to` | `string` | Client-side routing, used for navigating within the website. The baseUrl will be automatically prepended to this value. | Yes | N/A |
+| `href` | `string` | A full-page navigation, used for navigating outside of the website. **Only one of `to` or `href` should be used.** | Yes | N/A |
+| `prependBaseUrlToHref` | `boolean` | Prepends the baseUrl to `href` values. | No | `false` |
+| `position` | `'left'` / `'right'` | The side of the navbar this item should appear on. | No | `'left'` |
```

Comment:
I think TS notation is quite known now, eventually we'll generate those tables in the future so using `'left' | 'right'` seems appropriate